src/Hangfire.Tags.PostgreSql/TagsStorageTransactionRetryPolicy.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using Hangfire.Logging;
+using Hangfire.Tags.Storage;
+using Npgsql;
+
+namespace Hangfire.Tags.PostgreSql
+{
+    /// <summary>
+    /// Retry policy to address https://github.com/face-it/Hangfire.Tags/issues/35
+    /// </summary>
+    public class TagsStorageTransactionRetryPolicy : ITagsStorageTransactionRetryPolicy
```

Comment:
This class is intentionally made public so that consumers can increase the retry-count if needed.